src/App.js

```diff
@@ -34,7 +36,18 @@ const configValues = {
 setConfig(configValues);
 
 function App() {
-  return <PetsList />;
+  return (
+    <React.Fragment>
+      <Router>
+        <Route exact path="/" component={PetsList} />
+        <Route
+          exact
+          path="/petDetails/:animalId"
```

Comment:
@tmccoy529 just a reminder, unless we figure out how to improve the query status will likely need to be included in this. This is fine for now.

Reply:
actually this can probably just be `/pets/:id`

Reply:
done